### ISSUE TITLE
New smaller docker images do not compile plotters

### ISSUE THREAD

**Issue by abanchev** [guydavis/machinaris#465]
**Describe the bug**
Cmake is missing so plotman never works with latest builds as none of the plot managers get compiled correctly
apt does not work properly as repositories are not signed
W: GPG error: http://security.ubuntu.com/ubuntu hirsute-security InRelease: gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
E: The repository 'http://security.ubuntu.com/ubuntu hirsute-security InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**To Reproduce**

Steps to reproduce the behavior, e.g.:
1. Create a new docker image and check the logs, 

**Expected behavior**
Madmax and other plotters get built and can be started

**System setup:**
 - OS: Unraid
 - Docker version number (output of `docker info`)
 - Machinaris branch: test
 - Machinaris version number 0.6.7

**Comment by abanchev** [comment]
Here is the content of the madmax_setup.log

> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
> 
> Get:1 http://security.ubuntu.com/ubuntu hirsute-security InRelease [110 kB]
> Get:2 http://archive.ubuntu.com/ubuntu hirsute InRelease [269 kB]
> Get:3 https://mega.nz/linux/MEGAsync/xUbuntu_21.04 ./ InRelease [2441 B]
> Err:3 https://mega.nz/linux/MEGAsync/xUbuntu_21.04 ./ InRelease
>   gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> Err:1 http://security.ubuntu.com/ubuntu hirsute-security InRelease
>   gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> Err:2 http://archive.ubuntu.com/ubuntu hirsute InRelease
>   gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> Get:4 http://archive.ubuntu.com/ubuntu hirsute-updates InRelease [115 kB]
> Err:4 http://archive.ubuntu.com/ubuntu hirsute-updates InRelease
>   gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> Get:5 http://archive.ubuntu.com/ubuntu hirsute-backports InRelease [101 kB]
> Err:5 http://archive.ubuntu.com/ubuntu hirsute-backports InRelease
>   gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> Reading package lists...
> W: GPG error: https://mega.nz/linux/MEGAsync/xUbuntu_21.04 ./ InRelease: gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> E: The repository 'https://mega.nz/linux/MEGAsync/xUbuntu_21.04 ./ InRelease' is not signed.
> W: GPG error: http://security.ubuntu.com/ubuntu hirsute-security InRelease: gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> E: The repository 'http://security.ubuntu.com/ubuntu hirsute-security InRelease' is not signed.
> W: GPG error: http://archive.ubuntu.com/ubuntu hirsute InRelease: gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> E: The repository 'http://archive.ubuntu.com/ubuntu hirsute InRelease' is not signed.
> W: GPG error: http://archive.ubuntu.com/ubuntu hirsute-updates InRelease: gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> E: The repository 'http://archive.ubuntu.com/ubuntu hirsute-updates InRelease' is not signed.
> W: GPG error: http://archive.ubuntu.com/ubuntu hirsute-backports InRelease: gpgv, gpgv2 or gpgv1 required for verification, but neither seems installed
> E: The repository 'http://archive.ubuntu.com/ubuntu hirsute-backports InRelease' is not signed.
> Cloning into 'chia-plotter'...
> Building madmax on x86_64...
> Submodule 'lib/BLAKE3' (https://github.com/BLAKE3-team/BLAKE3.git) registered for path 'lib/BLAKE3'
> Submodule 'lib/bls-signatures' (https://github.com/Chia-Network/bls-signatures.git) registered for path 'lib/bls-signatures'
> Submodule 'lib/libbech32' (https://github.com/dcdpr/libbech32.git) registered for path 'lib/libbech32'
> Cloning into '/chia-plotter/lib/BLAKE3'...
> Cloning into '/chia-plotter/lib/bls-signatures'...
> Cloning into '/chia-plotter/lib/libbech32'...
> Submodule path 'lib/BLAKE3': checked out '037de38bfec4e813ab6189a50cb7c4cbae47268a'
> Submodule path 'lib/bls-signatures': checked out 'a9618ee02b085d9914b715a3fafd097c1c3791bd'
> Submodule path 'lib/libbech32': checked out '954b14fe81602dde0cf5cb6d45208403160fb76c'
> Note: switching to 'c99a449c0467923c576acffcb1b7152ff94bb289'.
> 
> You are in 'detached HEAD' state. You can look around, make experimental
> changes and commit them, and you can discard any commits you make in this
> state without impacting any branches by switching back to a branch.
> 
> If you want to create a new branch to retain commits you create, you may
> do so (now or later) by using -c with the switch command. Example:
> 
>   git switch -c <new-branch-name>
> 
> Or undo this operation with:
> 
>   git switch -
> 
> Turn off this advice by setting config variable advice.detachedHead to false
> 
> HEAD is now at c99a449 Merge pull request #978 from madMAx43v3r/chia-bls
> M       CMakeLists.txt
> ./make_devel.sh: line 7: cmake: command not found
> make: *** No targets specified and no makefile found.  Stop.
> cp: cannot stat './build/*': No such file or directory

**Comment by guydavis** [comment]
Yes, it looks like your ISP, or your own Pihole?, is choosing to block certain sites.  So, given your network rules, you'll likely have to try disabling: `blockchain_db_download`  Set it to false.  That will disable any attempt to reach mega.nz.

**Comment by abanchev** [comment]
Networking is good, ping is working to mega.nz, github and ubuntu.com (curl also works)
networking was host, bridge didn't change anything.

**Comment by guydavis** [comment]
Well that's an interesting issue you got there.  Have a look at `/machinaris/scripts/madmax_setup.sh` and others.  Do let me know what you find from your location. Good luck!

**Comment by abanchev** [comment]
https://stackoverflow.com/questions/66319610/gpg-error-in-ubuntu-21-04-after-second-apt-get-update-during-docker-build

The issue seems to be related to old version of docker, i am running 6.8.3 which may be the reason of the incompatibility

**Comment by guydavis** [comment]
Good to know there may be issues with old versions of Docker.  I'll move this to the Discussions area for others to benefit from.
